### PR TITLE
Add getChannelLink, getMessageLink, getUserLink

### PR DESCRIPTION
**Pull request by gunar**

### CODE REVIEW

File: src/slack/methods/format.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const { oneLineTrim } = require('common-tags')
+const { replace } = require('lodash/fp')
+
+const getChannelLink = (channelId, display) =>
```
Comment (keenahn) [inline src/slack/methods/format.js]: `({ channelId, display })`

File: src/slack/methods/format.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const removeDot = replace('.')('')
+const getMessageLink = ({ teamName, channelId, ts, display }) =>
+  (display
+    ? oneLineTrim`<https://${teamName}.slack.com/archives/
+        ${channelId}/p${removeDot(ts)}|${display}>`
+    : oneLineTrim`<https://${teamName}.slack.com/archives/
+        ${channelId}/p${removeDot(ts)}>`)
+
+const getUserLink = (userId, display) =>
```
Comment (keenahn) [inline src/slack/methods/format.js]: `({ userId, display })`

**Comment by keenahn** [comment]
1. These methods don't follow the pattern of accepting a `slack` object and returning a scoped method. I guess it makes sense since they don't actually make api calls, but then we should fix the `addMethods` method if that's the case. 
2. Naming methods `getxxx` implies that it is reading a value. I would call these `channelLink` `userLink` `messageLink` and/or have them under a `format` object (so it would be `slack.format.userLink({  ... }))`

**Comment by keenahn** [comment]
And needs tests foo

**Comment by gunar** [comment]
@keenahn Would you please implement (1) and (2) since you're move familiar with the code?

**Comment by keenahn** [comment]
I could do it for you but then you won't learn. Just look at my other functions

**Comment by gunar** [comment]
KTJ will refactor slack-wrap so I'm blocking this for now.

**Comment by gunar** [comment]
> https://gunargessner.slack.com/archives/C5FS003B2/p1508077812000009

**Pull request closed without merging**